testsuite/k8s-conformance-test-suite.sh

```diff
@@ -1,68 +1,142 @@
-# Set the following environment variables to run the test suite
-
-# Common Variables
-# Some of the variables need to be populated from the service principal and storage account details provided to you by Microsoft
-connectedClustedId=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 7 ; echo '')
-AZ_TENANT_ID= # tenant field of the service principal
-AZ_SUBSCRIPTION_ID= # subscription id of the azure subscription (will be provided)
-AZ_CLIENT_ID= # appid field of the service principal
-AZ_CLIENT_SECRET= # password field of the service principal
-AZ_STORAGE_ACCOUNT= # name of your storage account (will be provided)
-AZ_STORAGE_ACCOUNT_SAS="<your-sas-token-here>" # sas token for your storage account, please add it within the quotes (will be provided)
-RESOURCE_GROUP= # resource group name (will be provided)
-OFFERING_NAME= # name of the partner offering; use this variable to distinguish between the results tar for different offerings
-CLUSTERNAME=arc-partner-test-$connectedClustedId # name of the arc connected cluster
-LOCATION=eastus # location of the arc connected cluster
-
-# Platform Cleanup Plugin
-CLEANUP_TIMEOUT=1500 # time in seconds after which the platform cleanup plugin times out
-
-# In case your cluster is behind an outbound proxy, please add the following environment variables in the below command
-# --plugin-env azure-arc-platform.HTTPS_PROXY="http://<proxy ip>:<proxy port>"
-# --plugin-env azure-arc-platform.HTTP_PROXY="http://<proxy ip>:<proxy port>"
-# --plugin-env azure-arc-platform.NO_PROXY="kubernetes.default.svc,<ip CIDR etc>"
-
-# In case your outbound proxy is setup with certificate authentication, follow the below steps:
-# Create a Kubernetes generic secret with the name sonobuoy-proxy-cert with key proxycert in any namespace:
-# kubectl create secret generic sonobuoy-proxy-cert --from-file=proxycert=<path-to-cert-file>
-# By default we check for the secret in the default namespace. In case you have created the secret in some other namespace, please add the following variables in the sonobuoy run command: 
-# --plugin-env azure-arc-platform.PROXY_CERT_NAMESPACE="<namespace of sonobuoy secret>"
-# --plugin-env azure-arc-agent-cleanup.PROXY_CERT_NAMESPACE="namespace of sonobuoy secret"
+# read config file i.e., azure-arc-conformance.properties
+declare -A properties
+declare -A enabled_plugins
+
+properties_count=0
+plugin_count=0
+repository="https://github.com/santosh02iiit/azure-arc-validation.git"
+
+while IFS= read -r line; do
+    # remove leading and trailing whitespaces
+    line="$(echo -e "${line}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+
+    if [[ "${line}" == "#"* ]] #ignore lines starting with comments
+    then
+        continue
+    elif [ -z "${line}" ] #ignore empty lines
+    then
+        continue
+    fi
+
+    if [[ $line == *"enable"* ]] 
```

Comment:
we can change this check to $line == *"enable=true"

Reply:
we would also have to add a case to ignore $line == *"enable=false"

Reply:
modified it to *".enable="*